README.md

```diff
@@ -22,3 +22,34 @@ Run the following commands in the project directory:
 - Run ``npm run serve`` and open the displayed URL in your browser
 
 
+## Project Setup
+
+1. Clone the project from GitHub: `git clone <repository_url>`
+2. Open the project in Visual Studio Code.
+3. Deploy the contract.
+
+## Install Dependencies
+
+In the terminal of the Visual Studio Code console, follow these steps:
+
+1. Install Vue CLI globally: `npm install -g @vue/cli`
+2. Install web3: `npm install web3`
+3. Install big-integer: `npm install big-integer`
+
+## Setup MetaMask
+
+1. Download and install MetaMask.
+2. Connect MetaMask with Truffle by adding the Truffle address and chain ID:
+   - Network Name: ETH
+   - New RPC URL: http://127.0.0.1:8545
+   - Chain ID: 1337
+3. Import an account from Ganache into MetaMask.
+
+## Start Vue
```

Comment:
This section is also already covered. We could add the `npm run build` command to the front end, though.